editor/editor_properties.cpp

```diff
@@ -790,262 +790,293 @@ EditorPropertyFlags::EditorPropertyFlags() {
 }
 
 ///////////////////// LAYERS /////////////////////////
+void EditorPropertyLayersGrid::_rename_pressed(int p_menu) {
```

Comment:
Clearer with keeping the separation:
```suggestion
///////////////////// LAYERS /////////////////////////

void EditorPropertyLayersGrid::_rename_pressed(int p_menu) {
```